docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       - TZ=US/Central
 
   localstack:
-    image: localstack/localstack:1.2
+    image: localstack/localstack:1.3
```

Comment:
breaks local test - so updated to next version

more info here: https://pocket.slack.com/archives/C03QVL4SU/p1677016106673369